app/generated_resources.grd

```diff
@@ -6729,7 +6729,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
         Passwords are saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Brave Password Manager</ex></ph> on this device.
       </message>
       <message name="IDS_PASSWORD_GENERATION_PROMPT_GOOGLE_PASSWORD_MANAGER" desc="Autofill dropdown text describing password generation.">
-          You won't need to remember this password. It will be saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Brave Password Manager</ex></ph> for <ph name="EMAIL">$2<ex>user@gmail.com</ex></ph>.
+          You won't need to remember this password. It will be saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Brave Password Manager</ex></ph>.
```

Comment:
Unfortunately, this fix won't work. The file is auto-generated by our rebase scripts from Chromium's file and it will get overwritten on each Chromium update. To make this kind of change you'd need to add the fixed string to `app/brave_generated_resources.grd` and then add `chromium_src` overrides to every Chromium file where this string is used.
For an example, please see https://github.com/brave/brave-core/pull/30737